### PR TITLE
integration-and-controlling

### PR DESCRIPTION
**Pull request by SteveLim81**

### CODE REVIEW

**Comment by SteveLim81** [comment]
Need review

**Comment by SteveLim81** [comment]
relate the slack test

File: source/integration/integration-paloalto.rst [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+   -  `Deployment of GNAC using PAN Firewall`_
+   -  `Configuring PAN Firewall for integration via XML API`_
+   -  `Configuring PAN Firewall for Integration via SYSLOG`_
+
+
+**About this Guide**
+--------------------
+This guide describes how GNAC engineers and enterprise operators can work with GNAC and PAN firewall to send information of user authentication to PAN firewall.
+
+PAN Firewall generally require that when a user changes a department or location, the IP information changes and the assigned permissions are modified accordingly.
```
Comment (weatonusmc) [inline source/integration/integration-paloalto.rst]: requires

File: source/integration/integration-paloalto.rst [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+IP-based firewall policies do not know who is using IP, but they can work with GNAC to get user information about IP.
```
Comment (weatonusmc) [inline source/integration/integration-paloalto.rst]: an IP

File: source/integration/integration-paloalto.rst [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+Based on this information, even if the user's department or location is moved and the IP information is changed, the user will be able to apply the authority assigned to each user without modifying the rule in the firewall.
+Efficiently improves administrator's internal infrastructure operation and security.
+
+
+**Deployment of GNAC using PAN Firewall**
+-----------------------------------------
+GNAC provides the integration of authentication with Palo Alto via both way XML API and syslog protocol.
```
Comment (weatonusmc) [inline source/integration/integration-paloalto.rst]: "both way"  remove this as it is not needed.  "via XML API and Syslog protocol."

File: source/integration/integration-paloalto.rst [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+.. image:: /images/process.jpg
+   :width: 700px
+      
+   
+The authentication process is described below:
+
+#. User Authentication in GNAC
+#. GNAC sends authentication user and IP information to PAN firewall
+#. The PAN firewall compares the authentication user and IP information it receives from GNAC with its own user ID table.
+#. PAN confirm tag assigned to User-ID
```
Comment (weatonusmc) [inline source/integration/integration-paloalto.rst]: "confirm" replace with "confirms the"

**Comment by SteveLim81** [comment]
Can you review the created and changed file below that
- To change the description and number.
- Create the doc about "Query and delete DHCP Lease status"

File: source/integration/integration-paloalto.rst [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+   -  `2. Deployment of GNAC using PAN Firewall`_
+   -  `3. Configuring PAN Firewall for integration via XML API`_
+   -  `4. Configuring PAN Firewall for Integration via SYSLOG`_
+
+
+**1. About this Guide**
+-----------------------
+This guide describes how GNAC engineers and enterprise operators can work with GNAC and PAN firewall to send information of user authentication to PAN firewall.
+
+PAN Firewall generally require that when a user changes a department or location, the IP information changes and the assigned permissions are modified accordingly.
```
Comment (weatonusmc) [inline source/integration/integration-paloalto.rst]: require -> requires

**Comment by kyeyeon** [comment]
Need to fix
- follow page template (see authentication/enabling-authentication/8021x.html)
- place this doc somewhere under toctree (controlling network access)

**Pull request closed without merging**